devops/scripts/load_terraform_env.sh

```diff
@@ -10,9 +10,16 @@ if [ ! -f "$1" ]; then
     #exit
   fi
 else
-  # doesn't work with quotes
-  # shellcheck disable=SC2046
-  export $(grep -v -e '^[[:space:]]*$' -e '^#' "$1"  | sed 's/.*=/TF_VAR_\L&/' | xargs)
+  while read -r line
```

Comment:
This changes a "somewhat hard to understand" bit of script into a "even harder to understand" bit of script

What's the driver behind this? Also, is there any way we can combine the changes across both files so that we're not duplicating this? These two "load" scripts are very similar.

Reply:
I'll look add adding some comments to the script to explain it a bit more

Reply:
Update: I revisited and found a way to achieve this 😄